keyboards/speedo/readme.md

```diff
@@ -1,16 +1,24 @@
 # Speedo
 
-![Speedo](https://raw.githubusercontent.com/pcewing/speedo/master/build_log/images/04-03_23-09-56_00.jpg)
+![Speedo](http://assets.cozykeys.xyz/images/keyboards/speedo/speedo-v3.0-angle-led-front_1600x1600.png)
 
-Keyboard Maintainer: [Paul Ewing](https://github.com/pcewing)  
-Hardware Supported: Teensy 2.0  
-Hardware Availability: You will have to get the plates cut by an online service such as Ponoko. Case files are available in the [Speedo Repository](https://github.com/pcewing/speedo).
+- Keyboard Maintainer: [Paul Ewing](https://github.com/pcewing)
+- Hardware Supported: Elite-C
+- Hardware Availability: [Speedo Repository](https://github.com/cozykeys/speedo)
 
 Make example for this keyboard (after setting up your build environment):
 
     make speedo:default
 
-See [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
+Example of flashing this keyboard:
+
+    make speedo:default:dfu
+
+See [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools)
+then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide)
+for more information.
```

Comment:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Reply:
I generally like to keep text width at 80 whenever possible so that files are easier to work with in vim. These line breaks don't have make any visual difference in the file once rendered on Github so I'd prefer to keep the lines broken up.

If the main goal here is just to change to wording from `then` to `and` I'm happy to make that change.